packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -266,19 +265,13 @@ const subscriptionForId = (
   subscriptionId: string,
   subscriptions: SubscriptionsFetchState
 ): Subscription | null => {
-  if (subscriptions.result === null) {
-    return null;
-  }
-  return subscriptions.result.filter(
+  return (subscriptions.result as Subscription[]).filter(
     subscription => subscription.subscriptionId === subscriptionId
   )[0];
 };
 
 const planForId = (planId: string, plans: PlansFetchState): Plan | null => {
-  if (plans.result === null) {
-    return null;
-  }
-  return plans.result.filter(plan => plan.plan_id === planId)[0];
+  return (plans.result as Plan[]).filter(plan => plan.plan_id === planId)[0];
```

Comment:
Is this and the above change for subscriptions because `subscriptions.result` and `plans.result` can never be `null`?

Reply:
Yeah, basically... I thought they *could* be, but we short-circuit by displaying an error dialog when that happens and never get to this point. Explained another way, I find a code path where I could test this particular condition.